src/canvas-extractor/edfi_canvas_extractor/api/authentication_events.py

```diff
@@ -52,10 +55,10 @@ def custom_get_new_page(self: PaginatedList):
         try:
             data = data[self._root]
         except KeyError:
-            # TODO: Fix this message to make more sense to an end user.
             raise ValueError("Invalid root value specified.")
 
-    if "audit/authentication/users" in self._first_url:
+    # This is the change to the code, added to get to the events object in the response
+    if "events" in data:
```

Comment:
Since we are narrowing to only do this when pulling auth events, use the "events" object if it's there. This should handle Mario's cases from https://tracker.ed-fi.org/browse/LMS-265?focusedCommentId=62750&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-62750